message_definitions/v1.0/common.xml

```diff
@@ -925,14 +925,14 @@
         <param index="7">Empty</param>
       </entry>
       <entry value="94" name="MAV_CMD_NAV_PAYLOAD_PLACE">
-        <description>Descend and place payload.  Vehicle descends until it detects a hanging payload has reached the ground, the gripper is opened to release the payload</description>
-        <param index="1">Maximum distance to descend (meters)</param>
+        <description>Descend and place payload. Vehicle moves to specified location, descends until it detects a hanging payload has reached the ground, and then releases the payload. If ground is not detected before the reaching the maximum descent value (param1), the command will complete without releasing the payload.</description>
+        <param index="1" label="Max Descent" minValue="0" units="m">Maximum distance to descend.</param>
         <param index="2">Empty</param>
         <param index="3">Empty</param>
         <param index="4">Empty</param>
-        <param index="5">Latitude (deg * 1E7)</param>
-        <param index="6">Longitude (deg * 1E7)</param>
-        <param index="7">Altitude (meters)</param>
+        <param index="5">Latitude</param>
```

Comment:
These are all missing units.

Reply:
This is actually intentional - there isn't a label either. 

Specifically, an MAV_CMD with `hasLocation` or `isDestination` not explicitly set to `false` are assumed to be have params 5, 6, 7 map to lat, lon, altitude by the [schema](https://mavlink.io/en/guide/xml_schema.html#MAV_CMD).
That means the units of altitude are known, as are the labels of all of them.

However the units of lat or long cannot be known on command definition because that depends on the frame and message used to send them. Which is why "deg * 1E7" was removed.

Reply:
... I really don't like the hasDestination/hasLocation stuff. MAV_CMD is now a *extra* special enum, and all enums have to support the flag in a generator in order to support it on MAV_CMD, even though it has no meaning/doesn't map onto any other enum...

Reply:
@WickedShell I don't have an opinion, other than there was a discussion on this stuff, and as far as I know, no one said it was a bad idea at the time. Have a chat to @DonLakeFlyer to see if there is a better way?

Reply:
Oh it's totally valuable information, I just object to shoe horning all enums with it, when it only matters in the one case, and is only useful by knowing it's part of the extra special enum. (Particularly with the default to being true)

Reply:
Everything about MAV_CMD is extra special :-( 
Is there a solution to your problem that can also provide this information?